ws-nextjs-app/integration/pages/live/arabic/amp.test.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * @service arabic
+ * @pathname /arabic/live/67574192
```

Comment:
AMP is not supported for Live Pages, therefore this can be deleted

Reply:
Can the Pidgin amp tests file in that same location also be deleted then?

Reply:
Yes, we did a bit of digging in the code, and we found that the NextJS app does not currently support AMP pages. I would want some 👀 from the Content Team on this PR before we merge this, just to sense check our assumptions.